proposals/lip-0053.md

```diff
@@ -102,9 +101,9 @@ The role of relayer is totally symmetric for relaying information from a sidecha
 
 The Lisk protocol allows relayers to post cross-chain update transactions which do not contain all CCMs, as opposed to the mechanism explained in the previous section. This can be useful for example when the list of CCMs is too large to be included in a single block.
```

Comment:
This section is a bit of a mix of a concrete algorithm for relayers for creating a valid CCU, an explanation for the specifications and rationale for it. I think it would improve readability to separate these different aspects more clearly.

Reply:
As I mentioned in the stand-up, I prefer not to change aspects of the LIP that are not wrong or changes in the protocol.

Reply:
All good then.